frontend/src/styles/buttons/ScrollToTopButton.module.css

```diff
@@ -0,0 +1,23 @@
+.ScrollToTopButton {
+    position: fixed;
+    display: inline-block;
+    right: 2%;
+    bottom: 20%;

```

Comment:
possibly: this could be too much? consult katherine on design

Reply:
@heylookitsme can you elaborate on what is too much? are you referring to the bottom margin? If so, I believe that was an arbitrary value decided by me, but when I showcased it to the designers during the spring semester, I don't remember them commenting on that being an issue, but I guess it would be good to consule Katherine specifically about this again just in case